### PR TITLE
[RFC] topology: define a channel map macro for all topologies

### PR DESCRIPTION
**Pull request by juimonen**
Previously we had a channel map macro only in general dmic topology. As
the pipelines are used with dmic also used with other topologies the
dmic specific macro doesn't work anymore. Thus define generic macro in
pipeline.m4 that defaults to stereo channel map. This can be overridden
in topologies as shown in general dmic.

Signed-off-by: Jaska Uimonen <jaska.uimonen@linux.intel.com>

### CODE REVIEW

File: tools/topology/m4/pipeline.m4 [hunk 1]
```diff
@@ -7,6 +7,13 @@ define(`SCHEDULE_TIME_DOMAIN_TIMER', 1)
 # default number of DAI periods
 define(`DAI_DEFAULT_PERIODS', 2)
 
+# default stereo channel map
+ifdef(`VOL_CH_MAPS', , `define(`VOL_CH_MAPS', LIST(`	', KCONTROL_CHANNEL(FL, 1, 0),
+							   KCONTROL_CHANNEL(FR, 1, 1)))')
+
+ifdef(`SW_CH_MAPS', , `define(`SW_CH_MAPS', LIST(`	', KCONTROL_CHANNEL(FL, 2, 0),
+							   KCONTROL_CHANNEL(FR, 2, 1)))')
+
```
Comment (plbossart) [inline tools/topology/m4/pipeline.m4:16]: what does "SW" stand for? Software?  Switch?

Comment (juimonen) [reply tools/topology/m4/pipeline.m4:16]: yes SWITCH... I can rename it with longer and more clear name

**Review by singalsu — COMMENTED** [review]
Looks good and works correctly (just checked with X1). Please add Fixes: #4136 to commit message.

**Review by singalsu — APPROVED** [review]
Thanks Jaska!

**Comment by marc-hb** [comment]
Just a heads-up that Quickbuild is green but it did not actually run any tests: https://sof-ci.01.org/sof-pr-viewer/#/build/PR4142/build6395871 (internal issue PTS-31172)

Hopefully does not matter for this PR but something to keep in mind in general.

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**